### PR TITLE
Other: Update naming of UI components & commands.

### PR DESCRIPTION
**Pull request by jodator**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Other: Update naming of UI components & commands.

BREAKING CHANGE: Renamed `headings` drop-down UI component to `heading`.
BREAKING CHANGE: The "heading1", "heading2" and "heading3" commands are no longer available. Replaced by "heading" command that accepts heading model element name as a value.
BREAKING CHANGE: The `HeadingCommand#value` is no longer Boolean - only. Now it can have a name of heading model element.
BREAKING CHANGE: The `HeadingCommand` constructor's second parameter is now an array of supported model elements.

---

### Additional information

* PR of : https://github.com/ckeditor/ckeditor5-alignment/pull/25.
* Renamed 'headings' to 'heading'.
* Removed `headingn` commands in favor of `heading` one command.

### CODE REVIEW

**Comment by Reinmar** [comment]
Heading dropdown doesn't work.

**Comment by jodator** [comment]
Heading dropdown does work now.

**Comment by Reinmar** [comment]
Something got broken here:

image

**Comment by Reinmar** [comment]
Fancy heading isn't rendered as fancy.

File: src/headingcommand.js [hunk 1]
```diff
@@ -49,29 +51,37 @@ export default class HeadingCommand extends Command {
...
 	 * @fires execute
 	 */
-	execute() {
+	execute( options = {} ) {
 		const model = this.editor.model;
 		const document = model.document;
 
+		const modelElement = options.value;
+
+		if ( !this.modelElements.includes( modelElement ) ) {
```
Comment (Reinmar) [inline src/headingcommand.js]: Such checks cause silent errors. Throw explicitly or let app throw at some point.

**Comment by Reinmar** [comment]
> Fancy heading isn't rendered as fancy.

This is caused most likely by https://github.com/ckeditor/ckeditor5-engine/pull/1274, so I'll report a separate ticket for it.

I'll also report a separate ticket for these silent checks because they repeat in other packages.

File: src/headingcommand.js (hunk 1 above)
Comment (Reinmar) [reply src/headingcommand.js]: Reported a followup for this: https://github.com/ckeditor/ckeditor5/issues/890

**Pull request merged**